pkg/vgmanager/vgmanager_controller.go

```diff
@@ -249,6 +294,25 @@ func (r *VGReconciler) processDelete(ctx context.Context, volumeGroup *lvmv1alph
 		return nil
 	}
 
+	// Delete thin pool
```

Comment:
This should be done after checking if the volumegroup exists.

Reply:
That's right. Fixed.